packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/TorDropdown/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { Translation } from '@suite-components';
+import { Dropdown, DropdownRef, Icon, useTheme, variables } from '@trezor/components';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import { transparentize } from 'polished';
+import * as routerActions from '@suite-actions/routerActions';
+
+const Wrapper = styled.div<Pick<Props, 'marginLeft'>>`
+    ${props => props.marginLeft && `margin-left: 8px`};
+    position: relative;
+`;
+
+const TorStatus = styled.div`
+    display: flex;
+    cursor: default;
+    align-items: center;
+    & > * + * {
+        margin-left: 10px;
+    }
+`;
+
+const Indicator = styled.div<{ isActive?: boolean }>`
+    display: flex;
+    width: 24px;
+    height: 24px;
+    border-radius: 50%;
+    position: relative;
+    background: ${props =>
+        props.isActive
+            ? transparentize(0.9, props.theme.TYPE_GREEN)
+            : transparentize(0.9, props.theme.TYPE_LIGHT_GREY)};
+
+    &:after {
+        content: '';
+        position: absolute;
+        top: 8px;
+        left: 8px;
+        width: 8px;
+        height: 8px;
+        border-radius: 50%;
+        background: ${props =>
+            props.isActive ? props.theme.TYPE_GREEN : props.theme.TYPE_LIGHT_GREY};
+    }
+`;
+
+const Details = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Label = styled.div`
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_DARK_GREY};
+`;
+
+const Address = styled.div`
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+const GoToTorSettings = styled.button`
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    background: none;
+    border: 0;
+    width: 16px;
+    margin: 0 10px 0 auto;
+`;
+
+interface Props {
+    marginLeft?: boolean;
+    isActive?: boolean;
+}
+
+const TorDropdown = (props: Props) => {
+    const [open, setOpen] = useState(false);
+    const [torAddress, setTorAddress] = useState('');
+    const dropdownRef = useRef<DropdownRef>();
+    const theme = useTheme();
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+    useEffect(() => {
+        window.desktopApi?.getTorAddress().then(address => setTorAddress(address));
+    });
+
+    return (
+        <Wrapper {...props}>
+            <Dropdown
+                onToggle={() => setOpen(!open)}
+                ref={dropdownRef}
+                alignMenu="right"
+                offset={34}
+                horizontalPadding={12}
+                topPadding={12}
+                bottomPadding={12}
+                minWidth={230}
```

Comment:
```suggestion
                minWidth={230}
                onClick={() => goto('settings-index')}
```

This TOR dropdown will change right? However, right now it is extremely confusing for me.
I would make at least the whole dropdown clickable for now. Now if you clicked on different place than right arrow, it closes.